client/blocks/reader-full-post/_content.scss

```diff
@@ -405,7 +405,7 @@
 	color: #fff;
 }
 .wp-block-social-links.is-style-logos-only .wp-social-link-youtube {
-	color: red;
+	color: #f00;
```

Comment:
Just to note, `red / #f00` is the official YouTube red so not changing to variable here.